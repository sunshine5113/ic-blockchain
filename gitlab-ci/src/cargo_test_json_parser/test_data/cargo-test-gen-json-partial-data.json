{ "type": "suite", "event": "started", "test_count": 3 }
{ "type": "test", "event": "started", "name": "system_tests::request_signature_test" }
{ "type": "test", "name": "system_tests::request_signature_test::malicious_input_test", "event": "ok", "exec_time": "24.4760s"}
{ "type": "test", "name": "system_tests::request_signature_test::request_signature_test", "event": "ok", "exec_time": "1.4863s"}
{ "type": "test", "name": "system_tests::request_signature_test", "event": "ok", "exec_time": "26.0314s", "stderr": "Feb 15 10:19:30.860 INFO Running 'request_signature_test' with seed 42\nDEBG Using..."}
{ "type": "suite", "event": "ok", "passed": 3, "failed": 0, "ignored": 0 }
