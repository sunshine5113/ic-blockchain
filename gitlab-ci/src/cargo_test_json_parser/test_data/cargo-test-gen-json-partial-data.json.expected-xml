<?xml version="1.0" ?>
<testsuites disabled="0" errors="0" failures="0" tests="3" time="51.993700000000004">
	<testsuite disabled="0" errors="0" failures="0" name="cargo test" skipped="0" tests="3" time="51.993700000000004">
		<testcase name="system_tests::request_signature_test" time="26.031400" classname="system_tests">
			<system-err>Feb 15 10:19:30.860 INFO Running 'request_signature_test' with seed 42
DEBG Using...</system-err>
		</testcase>
		<testcase name="system_tests::request_signature_test::malicious_input_test" time="24.476000" classname="system_tests"/>
		<testcase name="system_tests::request_signature_test::request_signature_test" time="1.486300" classname="system_tests"/>
	</testsuite>
</testsuites>

