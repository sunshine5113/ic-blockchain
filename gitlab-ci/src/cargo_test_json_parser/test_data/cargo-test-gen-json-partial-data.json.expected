{"type": "suite", "event": "started", "test_count": 3}
{"type": "test", "event": "started", "name": "system_tests::request_signature_test"}
{"type": "test", "event": "ok", "name": "system_tests::request_signature_test", "exec_time": "26.0314s", "stderr": "Feb 15 10:19:30.860 INFO Running 'request_signature_test' with seed 42\nDEBG Using..."}
{"type": "test", "event": "started", "name": "system_tests::request_signature_test::malicious_input_test"}
{"type": "test", "event": "ok", "name": "system_tests::request_signature_test::malicious_input_test", "exec_time": "24.4760s"}
{"type": "test", "event": "started", "name": "system_tests::request_signature_test::request_signature_test"}
{"type": "test", "event": "ok", "name": "system_tests::request_signature_test::request_signature_test", "exec_time": "1.4863s"}
{"type": "suite", "event": "ok", "passed": 3, "failed": 0, "allowed_fail": 0, "ignored": 0, "measured": 0, "filtered_out": 0}
